src/pages/Mentors/index.tsx

```diff
@@ -31,13 +38,62 @@ const Mentors = () => {
   useEffect(() => {
     if (data) {
       const allMentors = data.pages.flatMap((page) => page.items);
-      setSortedMentors(allMentors);
+
+      const countries = allMentors
+        .map((mentor) => mentor.application?.country)
+        .filter((country): country is string => !!country);
+      setUniqueCountries([...new Set(countries)]);
+
+      const availableSlots = allMentors
+        .map((mentor) => {
```

Comment:
can you check mentor.application object for the availableSlots and approvedMenteesCount. It should be there. @rdwaynedehoedt